app/grandchallenge/archives/models.py

```diff
@@ -89,6 +89,12 @@ class Archive(UUIDModel, TitleSlugDescriptionModel, ViewContentMixin):
         blank=True,
         on_delete=models.SET_NULL,
     )
+    optional_hanging_protocols = models.ManyToManyField(
+        "hanging_protocols.HangingProtocol",
+        related_name="optional_hanging_protocols_of_archive",
+        blank=True,
+        help_text="Optional alternative hanging protocols for this archive",
```

Comment:
Why does this one have help text and the others not? Make it consistent.

Reply:
made them all consistent